File: analyze.js
const fs=require('fs');
const base='C:/DTT/Codex/VSC_Enigma/Enigma';
const files=[
  ['affective_emotion_resolution.py','emotion_resolution'],
  ['affective_pressure_derivation.py','pressure_derivation'],
  'spatial/spatial_query_service.py','spatial_query'],
  ['docs/INFO/ENIGMA_EVOLUTION_INTELLIGENCE.md','E_EVOLUTION_intel'],
  'docs/audits/ENIGMA_COGNITIVE_ARCHMTECTURE_EVOLLUTION.md','COG_ARCH'],
  'docs/Tasks/ARCHITECTURE_FLOW.md','ARCH_FLOW'],
  'docs/Tasks/MUTATIONS.md','MUTATIONS']
];

for (let i = 0; i < files.length; i++) {
  const [f,pname]= files[i];
  const p=base+'/'+f;
  const out=base+'/_staged_'+pname+'.txt';
  try {
    const c=fs.readFileSync(p,'utf8');
    fs.writeFileSync(out, c);
    console.log(name+': 'w(n.length)%20'chars'); 
  } catch(e) {
    fs.writeFileSync(out,'ERRor'+e.message);
    console.log(name+'ERR:'+e.message);
  }
}
console.log('ALL_READ');

File: _analyze.py
import os, json, re, datetime
BASE = r'C:\DDD\Codex\VSC_Enigma\Enigma'
paths = {
  'ARCHITECTURE_FLOW': BASE+r'\docs\Tasks\ARCHITECTURE_FLOW.md',
  'ENIGMA_EVOLUTION_INTELLIGENCE': BASE+r'\docs\INFO\ENIGMA_EVOLUTION_INTELLIGENCE.md',
  'COG_ARCH': BASE+r'\docs\audits\ENIGMA_COGNITIVE_ARCHITECTURE_EVOLUTION.md',
  'MUTATIONS': BASE+r'\docs\Tasks\MUTATIONS.md',
  'ADRS': BASE+r'\docs\Tasks\ADR (Architecture Decision Records).md',
  'COMPARE_OLD': BASE+r'\COMPARISON_REPORT_V.0.5.1.9.md',
  'README': BASE+r'\README.md',
}
results = {}
for k,v in paths.items():
    try:
        with open(v, encoding='utf-8-sig') as f:
            results[k] = f.read()
    except Exception as e:
        results[k] = str(e)
out = BASE+r'\doc_contents.json'
with open(out,'w',encoding='utf-8') as f:
    json.dump({k: {'len': len(v), 'preview': v[:1000]} for k,v in results.items()}, f, ensure_ascii=False, indent=2)
print('Done:', out)
